design/one-pager-ignore-changes.md

```diff
@@ -0,0 +1,354 @@
+which contains the resource state on the provider side and is being
+added alongside [ObserveOnly].
+
+In addition to the new management policy, we will also add a new field
+`spec.initProviders` which would contain parameters that should only be
+used at the creation time. This would allow users to specify fields that
+should be ignored on updates, but still be used during creation. This can
+be added as a separate feature.
+
+## Implementation
+
+### Management policy
+
+Proposed name for the new management policy is `PartialControl`, which
+would fit with the current `FullControl` and insinuate that the resource
+is not fully managed by Crossplane. Some other proposals for the naming
+are: `ExplicitControl`, `SelectiveControl`, `IgnoreLateInit`.
+
+The new management policy would be implemented in the `crossplane-runtime`
+and would just ignore the [ResourceLateInitialize] condition and 
+skip the managed resource update [here][Late Initialization Update] if
+the management policy is set to `PartialControl`.
+
+### Deletion policy
+
+`DeletionPolicy` was planned to be deprecated in favour of the new management
+policies according to [the ObserveOnly design doc.][ObserveOnly] However,
+with `PartialControl` it's not clear if it should delete or orphan the
+external resources. So to avoid creating `PartialControl` and
+`PartialControlOrphanOnDelete` management policies, we will keep the
+`DeletionPolicy` to be able to specify the behaviour. In that case, we will
+need to deprecate and remove the `OrphanOnDelete` management policy as it
+won't be needed anymore.
+
+So the management policies would be:
+- `FullControl`
+- `PartialControl`
+- `ObserveOnly`
+
+With a note that for `ObserveOnly` the `DeletionPolicy` will be ignored,
+as we are in read-only mode.
+
+### initProviders
+
+The idea here was to do something similar to K8s `initContainers` and add a
+field `initProvider` to the managed resources. The parameters under
+`initProvider` would be used only during creation. The `initProvider` would
+have the same schema as `forProvider`.
+
+```yaml
+apiVersion: eks.aws.crossplane.io/v1alpha1
+kind: NodeGroup
+metadata:
+  name: my-group
+spec:
+  initProvider:
+    scalingConfig:
+      desiredSize: 1
+  forProvider:
+    region: us-east-1
+    clusterNameRef:
+      name: sample-cluster
+    subnetRefs:
+      - name: sample-subnet1
+    nodeRoleRef:
+      name: somenoderole
+    scalingConfig:
+      maxSize: 5
+      minSize: 1
+    updateConfig:
+      maxUnavailablePercentage: 50
+      force: true
+  providerConfigRef:
+    name: example
+```
+
+This would allow users to specify fields that would only be used on creation,
+but not on updates.
+
+When it comes to the implementation, from Crossplane side we would need to
+change the API. But the handling would all need to be done on the 
+provider level. The `initProvider` fields should only be used in the `Create`
+step of the managed resource reconciliation loop.
```

Comment:
> from Crossplane side we would need to change the API

I don't follow. What changes would we need to make in Crossplane? Wouldn't this change be isolated to providers?

Reply:
Good catch, actually it should not touch Crossplane API at all as the initProviders are only handled in providers, removing that line.